### ISSUE TITLE
Add a redirect resource for `sdk.privacy-center.org/*/loader.js`

### ISSUE THREAD

**Issue by AdamWr** [AdguardTeam/Scriptlets#172]
Maybe we could add a redirect resource for cookie consent, if it's not out of scope.

For example, some websites require to accept cookie notice, otherwise elements like video players or third party comments widget are not loaded.

Examples:
https://github.com/AdguardTeam/AdguardFilters/issues/109203
https://github.com/AdguardTeam/AdguardFilters/issues/103764
https://github.com/AdguardTeam/AdguardFilters/issues/103677
https://github.com/AdguardTeam/AdguardFilters/issues/100875
https://github.com/AdguardTeam/AdguardFilters/issues/96839
https://github.com/AdguardTeam/AdguardFilters/issues/96148
https://github.com/AdguardTeam/AdguardFilters/issues/83131
https://github.com/AdguardTeam/AdguardFilters/issues/82374
https://github.com/AdguardTeam/AdguardFilters/issues/71356
https://github.com/AdguardTeam/AdguardFilters/issues/41267
https://github.com/uBlockOrigin/uAssets/issues/11224
https://github.com/uBlockOrigin/uAssets/issues/10991

If I'm not wrong, all these issues above could be fixed by something like that:

<details>

```js
(function () {
  const DidomiWrapper = {
    isConsentRequired: function () {
      return false;
    },
    getUserConsentStatusForPurpose: function () {
      return true;
    },
    getUserConsentStatus: function () {
      return true;
    },
    getUserStatus: function() {},
    getRequiredPurposes: function () {
      return [];
    },
    getUserConsentStatusForVendor: function () {},
    Purposes: {
      Cookies: 'cookies'
    },
    notice: {
      isVisible: function() {
         return false; 
        }
    },
    on: function() {},
    shouldConsentBeCollected: function() { 
      return false; 
    },
    getUserConsentStatusForAll: function() {},
    getObservableOnUserConsentStatusForVendor: function() {
      return new UserConsentStatusForVendor();
    },
  };
  
  function UserConsentStatusForVendor() {};
  UserConsentStatusForVendor.prototype.first = function() {
    return new UserConsentStatusForVendorSubscribe();
  };
  UserConsentStatusForVendor.prototype.filter = function() {
    return new UserConsentStatusForVendorSubscribe();
  };
  UserConsentStatusForVendor.prototype.subscribe = function() {};
  
  function UserConsentStatusForVendorSubscribe() {}
  UserConsentStatusForVendorSubscribe.prototype.filter = function() {
    return new UserConsentStatusForVendorSubscribe();
  };
  UserConsentStatusForVendorSubscribe.prototype.subscribe = function() {};
  
  window.Didomi = DidomiWrapper;

  const didomiOnReadyWrapper = {
    push: function (arg) {
      if (typeof arg !== "function") {
        return;
      }
      setTimeout(arg(Didomi));
      /*
        Maybe it would be better to run it after website was loaded,
         otherwise in some cases it may not work correctly, for example:
         window.addEventListener('load', function () {
            setTimeout(arg(Didomi));
        });
      */
    }
  };
  window.didomiOnReady = didomiOnReadyWrapper;

  const didomiStateWrapper = {
    didomiGDPRApplies: "",
    didomiIABConsent: "",
    didomiVendorsConsent: "",
    didomiVendorsConsentUnknown: "",
    didomiVendorsConsentDenied: "",
    didomiPurposesConsent: "",
    didomiPurposesConsentUnknown: "",
    didomiPurposesConsentDenied: "",
    didomiExperimentId: "",
    didomiExperimentUserGroup: ""
  };
  window.didomiState = didomiStateWrapper;

  const tcData = {
    eventStatus: "tcloaded",
    gdprApplies: false,
    listenerId: function () {},
    vendor: {
      consents: []
    },
    purpose: {
      consents: []
    }
  };
  const __tcfapiWrapper = function (...args) {
    for (const arg of args) {
      if (typeof arg === "function") {
        try {
          setTimeout(arg(tcData, success = true));
        } catch (ex) {
          console.log(ex);
        }
      }
    }
  };
  window.__tcfapi = __tcfapiWrapper;
})();
```

</details>

**Comment by slavaleleka** [comment]
I see that sometimes unblocking `loader.js` is not enough so there are such rules in filters:
```
@@||sdk.privacy-center.org/*/loader.js
@@||sdk.privacy-center.org/sdk.*.js
```

and sometimes even:
```
@@||sdk.privacy-center.org/*/loader.js
@@||sdk.privacy-center.org/sdk.*.js
@@||sdk.privacy-center.org/ui-gdpr-*.js
```

Don't we need redirects for these other scripts?

**Comment by AdamWr** [comment]
I guess it may depends on case, but for all cases above, redirect resource for `loader.js` should be enough.

> and sometimes even:
> 
> ```
> @@||sdk.privacy-center.org/*/loader.js
> @@||sdk.privacy-center.org/sdk.*.js
> @@||sdk.privacy-center.org/ui-gdpr-*.js
> ```

As far as I can see, it's for example, for this issue - https://github.com/AdguardTeam/AdguardFilters/issues/71356 and it looks like that it could be fixed by redirecting just `loader.js`.
